procedures/release-duty/index.rst

```diff
@@ -204,17 +203,15 @@ These bookmarklets should help you view tasks and taskgroups in Firefox.
 *  Name: ``task inspector`` Location: `https://tools.taskcluster.net/tasks/%s <https://tools.taskcluster.net/tasks/%s>`__ ; Keyword: ``task``
 *  Name: ``taskgroup inspector`` Location: `https://tools.taskcluster.net/groups/%s <https://tools.taskcluster.net/groups/%s>`__ ; Keyword: ``taskgroup``
 *  Name: ``stop`` Location: ``javascript:stop();``
-   *  This can be used to stop further loading in the Task Group Inspector. It shouldn't be used when actively monitoring (i.e.: watching for failures), but it can greatly speed things up if you're using it for other reasons. Be sure to wait for the initial tasks to load before you use it.
+  *  This can be used to stop further loading in the Task Group Inspector. It shouldn't be used when actively monitoring (i.e.: watching for failures), but it can greatly speed things up if you're using it for other reasons. Be sure to wait for the initial tasks to load before you use it.
 
 Now if you go to your URL bar, you can type ``task TASKID`` or ``taskgroup TASKGROUPID`` and you'll go to that task or
 taskgroup in the inspector.
 
 After ReleaseDuty
 -----------------
 
-After your tour of releaseduty, it's customary to spend 1-2 weeks fixing release automation issues. Check the `Release
-Automation Improvements trello board <https://trello.com/b/BqnBcfXX/release-automation-improvements>`__ trello board for
-ideas of what to work on and to add new items as you discover them.
+After your tour of releaseduty, it's customary to fix any documentation or automation issues discovered.
```

Comment:
You don't necessarily have to fix all of them, especially lower-priority issues. We should make sure we file issues, at least.